plugins/enthusiast-common/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "enthusiast-common"
-version = "1.1.0"
+version = "1.2.0"
```

Comment:
I'd wait with bumping the version of the common package, until we create a new release - to keep it inline with project's version